lib/gooddata/models/project.rb

```diff
@@ -517,8 +525,21 @@ def transfer_schedules(from_project, to_project)
             if process_spec.type != :dataload
               executable = schedule_spec[:executable] || (process_spec.type == :ruby ? 'main.rb' : 'main.grf')
             end
+
             params = schedule_parameters(schedule_spec)
-            created_schedule = remote_process.create_schedule(schedule_spec[:cron] || schedule_cache[schedule_spec[:after]], executable, params)
+
+            if trigger_schedule
```

Comment:
the variable trigger_schedule is confusing, why trigger_schedule but updating trigger info = null.
We should rename it to: lazy_update_trigger_info

Reply:
Done update